game-server/data/handlers/ai/instance/empyreanCrucible/EmpyreanArbiterAI.java

```diff
@@ -71,15 +50,8 @@ public boolean onDialogSelect(Player player, int dialogActionId, int questId, in
 				}
 			}
 			PacketSendUtility.sendPacket(player, new SM_DIALOG_WINDOW(getObjectId(), 0));
-
-			getPosition().getWorldMapInstance().forEachPlayer(new Consumer<Player>() {
-
-				@Override
-				public void accept(Player p) {
-					PacketSendUtility.sendPacket(p, SM_SYSTEM_MESSAGE.STR_MSG_FRIENDLY_MOVE_COMBATAREA_IDARENA(player.getName()));
-				}
-
-			});
+			getPosition().getWorldMapInstance().forEachPlayer(
+				p -> PacketSendUtility.sendPacket(p, SM_SYSTEM_MESSAGE.STR_MSG_FRIENDLY_MOVE_COMBATAREA_IDARENA(player.getName())));
```

Comment:
Use `PacketSendUtility#broadcastToMap` instead of explicitly iterating over all players.

Reply:
`
int dialogId = player.getInventory().getFirstItemByItemId(186000124) != null ? 1011 : 0;
PacketSendUtility.broadcastToMap(player, new SM_DIALOG_WINDOW(getObjectId(), dialogId));
`